### ISSUE TITLE
Ability to run synchronously requestConsentInfoUpdate method

### ISSUE THREAD

**Issue by kenumir** [googleads/googleads-consent-sdk-android#38]
It would be nice to be able to run `requestConsentInfoUpdate` synchronously in my own thread with other blocking operations. The method should return `ConsentStatus` or throw exception (no internet etc)

**Comment by tgvoskuilen** [comment]
Agreed, or at least to be able to set a timeout on the http request. I had a few users in Asia complain that the app was hanging because the Consent SDK AsyncTask apparently hung, which blocked other AsyncTasks that my app uses during loading.
